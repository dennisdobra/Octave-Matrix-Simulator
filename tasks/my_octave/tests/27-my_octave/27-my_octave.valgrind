==62554== Memcheck, a memory error detector
==62554== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==62554== Using Valgrind-3.18.1 and LibVEX; rerun with -h for copyright info
==62554== Command: /home/dominic/Documents/Facultate_sem_3/PCLP1_asist/check_my_octave_pclp1_2023_v1.1/check_my_octave/my_octave
==62554== Parent PID: 62553
==62554== 
==62554== 
==62554== HEAP SUMMARY:
==62554==     in use at exit: 0 bytes in 0 blocks
==62554==   total heap usage: 405,687 allocs, 405,687 frees, 3,050,512 bytes allocated
==62554== 
==62554== All heap blocks were freed -- no leaks are possible
==62554== 
==62554== For lists of detected and suppressed errors, rerun with: -s
==62554== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
