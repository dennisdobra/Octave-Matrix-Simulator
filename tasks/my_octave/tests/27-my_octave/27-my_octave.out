Unrecognized command
Unrecognized command
Unrecognized command
Unrecognized command
No matrix with the given index
No matrix with the given index
